qcfractal/interface/models/torsiondrive.py

```diff
@@ -186,18 +185,23 @@ def get_final_molecules(self, key: Union[int, Tuple[int, ...], str]=None) -> Dic
         key : Union[int, str, None], optional
             Specifies a single entry to pull from.
 
-
         Returns
         -------
-        energy : Dict[str, Any]
+        final_molecules : Dict[str, Any]
             Returns molecule at each grid point in a dictionary or at a
             single point if a key is specified.
 
         Examples
         --------
+        >>> torsiondrive_obj.get_final_molecules()
+        {(-90,): molecule1, (-60,): molecule2, ...}
+
+        >>> torsiondrive_obj.get_final_molecules((-90,))
+        molecule1
+
+        >>> type(molecule1)
+        qcelemental.models.molecule.Molecule
 
-        >>> torsiondrive_obj.get_final_energies()
-        {(-90,):{'symbols': ['H', 'O', 'O', 'H'], 'geometry': [1.72669422, 1.28135788, ... }
```

Comment:
I am not sure the new example strings more clear. Maybe something like:

```py
>>> mols = torsiondrive_obj.get_final_molecules()
>>> type(mols[(-90, )])
qcelemental.models.molecule.Molecule
	
>>> type(torsiondrive_obj.get_final_molecules((-90,)))
qcelemental.models.molecule.Molecule
```

Reply:
Thanks! I like that it shows the general structure, but I agree it may be a little abstract. I have updated the example strings as you suggested.